Readability.js

```diff
@@ -544,17 +563,17 @@ Readability.prototype = {
        * A score is determined by things like number of commas, class names, etc. Maybe eventually link density.
       **/
       var candidates = [];
-      for (var pt = 0; pt < elementsToScore.length; pt += 1) {
-        var parentNode = elementsToScore[pt].parentNode;
+      this._forEachNode(elementsToScore, function(elementToScore) {
```

Comment:
So, here's something which I've been wondering about... are you trying to be consistent for its own sake? Because you could just call `elementsToScore.forEach(...)`, right? Or update `_forEachNode` to check `Array.isArray` before doing the `.call` thing...